Uno.Gallery/Uno.Gallery.UWP/Views/GeneralPages/OverviewPage.xaml

```diff
@@ -31,17 +32,27 @@
 					<local:OverviewSampleView SamplePageType="samples:ButtonSamplePage">
 						<StackPanel Spacing="8">
 
-							<Button Content="CONTAINED"
-									AutomationProperties.AutomationId="MaterialContainedButton"
-									Style="{StaticResource MaterialContainedButtonStyle}" />
-
+							<Button Content="ELEVATED"
```

Comment:
If I'm being really picky these should no longer be all uppercase. Material 3 removes the Button text capitalization stuff. Shouldn't even need this for M2 come to think of it, this should be taken care of by the ToUpperConverter in the M2 Button styles